core/src/integration-test/java/com/scalar/db/api/DistributedTransactionAdminIntegrationTestBase.java

```diff
@@ -131,6 +139,21 @@ private void dropTables() throws ExecutionException {
     admin.dropCoordinatorTables();
   }
 
+  @Test
+  public void
+      getTableMetadata_WhenDebugging_ShouldReturnCorrectMetadataWithTransactionMetadataColumns()
```

Comment:
If I move the test to `ConsensusCommitAdminIntegrationTestBase`, the test won't be available from the server integration test since there is only a `DistributedTransactionAdminServiceIntegrationTest` which inherits from `DistributedTransactionAdminIntegrationTestBase`. 
Should we add a new class `ConsensusCommitAdminIntegrationTestWithDistributedStorageService` which inherits from `ConsensusCommitAdminIntegrationTestBase` mainly for testing `getTableMetadata_WhenDebugging_ShouldReturnCorrectMetadataWithTransactionMetadataColumns` or is it too much ?

Reply:
Ah okay, that's good point. If we add the new class `ConsensusCommitAdminIntegrationTestWithDistributedStorageService`, we have a lot of duplicated tests between it and `DistributedTransactionAdminServiceIntegrationTest`.

IMO, for now, we can add the new class `ConsensusCommitAdminIntegrationTestWithDistributedStorageService` in this PR, and after that we should refactor and reorganize the integration tests in the `server` module. What do you think?

Reply:
Ok, I will add `ConsensusCommitAdminIntegrationTestWithDistributedStorageService`. Ok, let's talk about the test reorganization on the server side afterward.